db/migrate/20201021083148_change_uniq_on_profiles.rb

```diff
@@ -0,0 +1,8 @@
+class ChangeUniqOnProfiles < ActiveRecord::Migration[5.2]
+  def change
+    remove_index :profiles, name: 'uniqueness'
+    add_index :profiles,
+              %i[ref_id account_id benchmark_id external policy_id],
+              unique: true, name: 'uniqueness'
+  end
```

Comment:
We should write separate up/down migrations for this because `remove_index` only by name doesn't have a way to go back `down`.

Reply:
I'll add it. Reverting it once "external" profiles are present in Policies would not be possible if there would be a duplicate profile outside of policy.

Reply:
Good point!